lib/util/docsUrl.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+function docsUrl(ruleName) {
+  return `https://github.com/yannickcr/eslint-plugin-react/tree/master/docs/rules/${ruleName}.md`;
+}
+
+module.exports = {
+  docsUrl
```

Comment:
this should just be `module.exports = docsUrl`

Reply:
Fixed.